rio/src/RIO/Process.hs

```diff
@@ -599,6 +629,22 @@ findExecutable name = do
             return epath
     return epath
 
+-- | Test whether a file has executable permissions.
+isExecutable :: FilePath -> IO Bool
```

Comment:
I think best would be to include a `Bool` argument here indicating if it's Windows or not.

Reply:
Understood.